### ISSUE TITLE
Show the top two most important features in the IFE plot

### ISSUE THREAD

**Issue by mmahmoudian** [mmahmoudian/sivs#8]
At the moment, the IFE (iterative Feature Elimination) plot does not contain the name of the features with the highest VIMP. Although it is logical not to have them (it is useless to eliminate the second feature and train a model based on only one single feature), but it can get confusing for users who are not paying attention and are using SIVS blindly:

![image](https://github.com/mmahmoudian/sivs/assets/390889/46f5ab59-0e49-45ac-b78d-38680f15ddd9)

Nevertheless, as people are people, and there is a chance that they report important features based on what they see in the plot rather than `sivs_obj$vimp` or `sivs::suggest(sivs_obj)`, it is a good idea to somehow present these two features in the plot.

In other words we are trying to a [PEBCAK](https://en.wiktionary.org/wiki/PEBCAK) in the software.
